src/app/subscriptions/data/index.ts

```diff
@@ -49,3 +94,4 @@ export const SubscriptionCollection = {
     }
   },
 }
+export type SubscriptionCollection = ReturnType<typeof SubscriptionCollection.fromArray>
```

Comment:
Changes here to our transformation layer, partly an upgrade to the style here, but also brings parts of KDS and DiscoverySubscriptions together so we can just refer to a `Subscription` instead of a union of the two.